### PR TITLE
Create queries

### PR DESCRIPTION
**Pull request by kirtan96**
Added the getRoomData query
Swapped the startTime, endTime value

### CODE REVIEW

File: server/src/mocks/index.js [hunk 1]
```diff
@@ -58,11 +59,11 @@ export default async () => {
       const date = Math.round(Date.now() / 1000);
       await Event.create({
         createdBy: admin._id,
-        endTime: date,
```
Comment (kirtan96) [inline server/src/mocks/index.js:61]: Swapped it because endTime was smaller than startTime

File: server/src/graphql/resolvers/room-resolvers.js [hunk 2]
```diff
@@ -1,16 +1,34 @@
...
       // Generate RoomData
-      // return await Room.findBy({name: {$in: names}});
+
+      const room = await Room.findOne({
+        location: {$eq: roomID}
+      });
+
+      const config = await Config.findOne({
+        isActive: {$eq: true},
+        area: {$eq: room.area._id}
```
Comment (msalia) [inline server/src/graphql/resolvers/room-resolvers.js:17]: This just needs to be `room.area`

File: server/src/graphql/resolvers/room-resolvers.js [hunk 3]
```diff
@@ -1,16 +1,34 @@
...
+      });
+      const logo = config.logo;
+      const roomName = config.roomMapping.filter(data =>
+        data.room.toString() === room._id.toString())[0].name;
```
Comment (msalia) [inline server/src/graphql/resolvers/room-resolvers.js:21]: data.room === room._id

Comment (kirtan96) [reply server/src/graphql/resolvers/room-resolvers.js:21]: Tried it. Didn't work for some reason

File: server/src/graphql/resolvers/room-resolvers.js (hunk 2 above)
Comment (kirtan96) [reply server/src/graphql/resolvers/room-resolvers.js:17]: Nope..doesn't work without '_id'.

**Review by msalia — APPROVED** [review]

**Pull request merged**